webpack/assets/javascripts/react_app/redux/reducers/toasts/index.js

```diff
@@ -0,0 +1,24 @@
+import { TOASTS_ADD, TOASTS_DELETE } from '../../consts';
+import Immutable from 'seamless-immutable';
+
+const initialState = Immutable({
+  messages: {}
+});
+
+export default (state = initialState, action) => {
+  const { payload } = action;
+
+  switch (action.type) {
+    case TOASTS_ADD: {
+      return state.setIn(['messages', payload.key], payload.message);
```

Comment:
I'm just concerned that we may have other messages in the store in the future and want to prevent possible confusion.